packages/back-end/src/services/experiments.ts

```diff
@@ -2812,14 +2812,24 @@ export async function updateExperimentAnalysisSummary({
   const overallTraffic = experimentSnapshot.health?.traffic?.overall;
   const snapshotHealthPower = experimentSnapshot.health?.power;
 
-  const totalUsers = overallTraffic?.variationUnits?.reduce(
-    (acc, a) => acc + a,
-    0
-  );
+  const standardSnapshot =
+    experimentSnapshot.type === "standard" &&
+    experimentSnapshot.analyses?.[0]?.results?.length === 1;
+  const totalUsers =
+    (overallTraffic?.variationUnits.length
+      ? overallTraffic.variationUnits.reduce((acc, a) => acc + a, 0)
+      : standardSnapshot
+      ? // fall back to first result for standard snapshots if overall traffic
+        // is missing
+        experimentSnapshot?.analyses?.[0]?.results?.[0]?.variations?.reduce(
+          (acc, a) => acc + a.users,
+          0
+        )
+      : null) ?? null;
 
   const srm = getSRMValue(experiment.type ?? "standard", experimentSnapshot);
 
-  if (overallTraffic && totalUsers !== undefined && srm !== undefined) {
+  if (overallTraffic && srm !== undefined) {
```

Comment:
question: Does this (overallTraffic being defined) mean that we are still only updating `analysisSummary.health` (where both SRM & Multiple Exposures live) if we have executed the health query?

Shouldn't we remove this check given that we can get SRM & Multiple Exposures even without the Health query?

Maybe I am misunderstanding something.

Reply:
Oh that's a good call. What's happening is that due to mongoose or something else, I'm not sure, this field does always exist, it's just empty. I'll remove that check in my notifications PR, thank you.